shaders/Membrane.gdshader

```diff
@@ -63,9 +64,18 @@ float fresnel(float power, vec3 normal, vec3 view)
 void fragment(){
     vec4 albedo = texture(albedoTexture, UV);
 
-    vec4 normalmap = texture(normalTexture, UV2);
-    NORMAL_MAP = normalmap.xyz;
-    NORMAL_MAP_DEPTH = 0.5f;
+    vec3 normalmap1 = texture(normalTexture, UV).rgb;
+    vec3 normalmap2 = texture(wiggleNormalTexture, UV).rgb;
+    float blend_amount = 0.45f;
+
+    vec3 a = normalmap1*2.0 - 1.0;
+    vec3 b = normalmap2*2.0 - 1.0;
```

Comment:
```suggestion
    vec3 a = normalmap1 * 2.0f - 1.0f;
    vec3 b = normalmap2 * 2.0f - 1.0f;
```
Spaces should be used around operators to improve readability. Also shaders should really work with floats so numbers should be written preferrably with the `f` suffix (though I'm not always in the mood to nitpick about this).